File: src/CriadorBaseDados/Model/DB/MotivoSituacao.cs
using Realms;
using System;
using System.Collections.Generic;
using System.Text;

namespace CriadorBaseDados.Model.DB
{
    public class MotivoSituacao : RealmObject
    {
        [PrimaryKey]
        public int Cod { get; set; }
        public string Descricao { get; set; }
    }
}

File: src/Mobile/b2b/lib/model/core/motivo_situacao.dart
class MotivoSituacao {
  final int cod;
  final String descricao;

  MotivoSituacao(this.cod, this.descricao);

  MotivoSituacao.fromJson(Map<String, dynamic> json)
      : cod = json['cod'],
        descricao = json['descricao'];

  Map<String, dynamic> toMap() => <String, dynamic>{'cod': cod, 'descricao': descricao};
}
